src/main/flight/gps_rescue.c

```diff
@@ -142,11 +142,11 @@ void gpsRescueInit(void)
     rescueState.sensor.gpsRescueTaskIntervalSeconds = HZ_TO_INTERVAL(TASK_GPS_RESCUE_RATE_HZ);
 
     float cutoffHz, gain;
-    cutoffHz = positionConfig()->altitude_d_lpf / 100.0f;
+    cutoffHz = positionConfig()->altitude_d_lpf * .01f;
```

Comment:
Is this change intended ? Or should it be `* .001f`

Reply:
No, I think it's good. /100 equals to *0.01

Reply:
This is init code, so speed is not important here. Doesn't matter if it's `*0.01f` or `/100`

Reply:
Yes somehow I saw .01f not as 0.01 but 0.1 - so please ignore as @ctzsnooze already pointed out my misinterpretation.